docs/modules/automation-testing/pages/scripting/add-action-groups-in-appium-script.adoc

```diff
@@ -0,0 +1,144 @@
+= Add action groups in Appium script
+:navtitle: Add action groups in Appium script
+
+To enhance test analysis, you can group related Appium actions. Grouping actions can help in troubleshooting by marking specific phases of a test and aligning them with your testing objectives.
+
+== Action group Appium setting
+
+To add actions within your script, use the below Appium setting to assign actions to a specific group.
+
+[options="header"]
+|=======================
+|Appium setting | Description | Default value
+| `kobiton:stepGroup` | Set to any string value to assign actions to a group with the value as name. | `''` (blank)
```

Comment:
To make a little more clear, can we say "Set this field to any string value to assign actions to a group, using that value as the group name."